tuned/plugins/plugin_video.py

```diff
@@ -69,20 +108,53 @@ def _instance_init(self, instance):
 	def _instance_cleanup(self, instance):
 		pass
 
-	def _radeon_powersave_files(self, device):
+	def _files(self, device):
 		return {
 			"method" : "/sys/class/drm/%s/device/power_method" % device,
 			"profile": "/sys/class/drm/%s/device/power_profile" % device,
-			"dpm_state": "/sys/class/drm/%s/device/power_dpm_state" % device
+			"dpm_state": "/sys/class/drm/%s/device/power_dpm_state" % device,
+			"panel_power_savings": "/sys/class/drm/%s/amdgpu/panel_power_savings" % device,
 		}
 
+	def apply_panel_power_saving_target(self, device, sim=False):
+		"""Apply the target value to the panel_power_savings file if it doesn't already have it"""
+
+		# if we don't have a proxy, we can't tell if we're on battery
+		if not self.proxy:
+			self.setup_battery_signaling()
+			return None
+
+		# if we don't have the file, we might be radeon not amdgpu
+		if not os.path.exists(self._files(device)["panel_power_savings"]):
+			return None
```

Comment:
I would swap the first two checks to make sure we don't setup the signaling at all if no device supports `panel_power_savings`: 
```suggestion
		# if we don't have the file, we might be radeon not amdgpu
		if not os.path.exists(self._files(device)["panel_power_savings"]):
			return None

		# if we don't have a proxy, we can't tell if we're on battery
		if not self.proxy:
			self.setup_battery_signaling()
			return None
```